tools/EventClients/Clients/KodiSend/kodi-send.py

```diff
@@ -60,6 +67,7 @@ def main():
     ip = "localhost"
     port = 9777
     keymap = "KB"
+    loglevel = 0
```

Comment:
Would be nice to have a comment describing what level 0 represents, or just define LOG_LEVEL_X on the top of the file (to avoid the magic number)